runner/src/server/plugins/engine/models/submission/WebhookModel.ts

```diff
@@ -0,0 +1,76 @@
+import { DetailItem } from "../types";
+import { format } from "date-fns";
+import config from "server/config";
+
+function answerFromDetailItem(item) {
+  switch (item.dataType) {
+    case "list":
+      return item.rawValue;
+    case "date":
+      return format(new Date(item.rawValue), "yyyy-MM-dd");
+    case "monthYear":
+      const [month, year] = Object.values(item.rawValue);
+      return `${year}-${month}`;
```

Comment:
rawValue is an array of numbers so this could result in '2021-1', needs to be yyyy-MM.